nemo/collections/llm/gpt/data/pre_training.py

```diff
@@ -92,8 +106,12 @@ def setup(self, stage: str = "") -> None:
         num_test_samples = int(test_iters * self.data_sampler.global_batch_size)
 
         if self.trainer.limit_val_batches <= 1.0 and isinstance(self.trainer.limit_val_batches, float):
+            assert (
+                "blend" not in self.build_kwargs
+            ), f"When using a single data distribution, limit_val_batches <= 1.0 is not supported."
+
```

Comment:
Also I think this comment can be confusing to users. Can we elaborate what it means by single data distribution. Basically clarify that there is no split of "train", "valid", "test" in the dataset passed and float limit_val_batches isn't supported in that case.